ui/src/app/views/account-creation/component.ts

```diff
@@ -134,14 +140,14 @@ export class AccountCreationComponent implements AfterViewInit {
   }
 
   get showUsernameValidationError(): boolean {
-    if (isBlank(this.profile.username) || !this.usernameOffFocus) {
+    if (isBlank(this.profile.username) || this.usernameCheckInProgress) {
       return false;
     }
     return this.isUsernameValidationError;
   }
 
   get showUsernameValidationSuccess(): boolean {
-    if (isBlank(this.profile.username) || !this.usernameOffFocus) {
+    if (isBlank(this.profile.username) || !this.usernameOffFocus || this.usernameCheckInProgress) {
```

Comment:
nit: As this is now the only usage of this `usernameOffFocus`, I would probably invert it and rename to `usernameFocused` to make this a bit more readable.